WiseUnpacker/SabreTools.Serialization.Deserializers/WiseSectionHeader.cs

```diff
@@ -34,15 +36,11 @@ public class WiseSectionHeader : BaseBinaryDeserializer<SectionHeader>
                     return null;
 
                 // First executable file
-                if (header.FirstExecutableFileEntryLength == 0)
-                    return null;
-                else if (header.FirstExecutableFileEntryLength >= data.Length)
+                if (header.FirstExecutableFileEntryLength >= data.Length)
```

Comment:
Was there a reason that you removed the checks for length 0? That seemed like a reasonable check to have.

Reply:
The only value that is ever guaranteed in the header is the size of the main msi installer, all other values can and often are 0. If I returned on the size of the first executable being 0, a bunch of valid installers would fail